playbooks/UPLOAD_FILE_TO_CLOUD_STORAGE.md

```diff
@@ -0,0 +1,53 @@
+# cloud.gcp_ops.upload_file
+
+A playbook to upload a file from a remote host to GCP Cloud Storage.
+
+## Requirements
+
+GCP user account with at minimum the storage.objectCreator IAM role or equivalent permissions.
+
+## Playbook Variables
+
+* **host_with_file**: Name of the remote host with file to upload to. Default: `"all"`.
```

Comment:
why not use the same naming convention for this parameter? using something like ``upload_file_to_cloud_storage_host_name_or_group``
Note that this will also work if a host group is specified not only with hostname, this needs to be added in the description